framework/source/class/qx/ui/form/core/VirtualDropDownList.js

```diff
@@ -393,15 +393,7 @@ qx.Class.define("qx.ui.form.core.VirtualDropDownList",
       }
       else
       {
-        var nativeArray = target.toArray();
-        qx.lang.Array.removeAll(nativeArray);
-        for (var i = 0; i < source.getLength(); i++) {
-          nativeArray.push(source.getItem(i));
-        }
-        target.length = nativeArray.length;
-
-        // necessary for keeping preselected items in sync
-        target.fireDataEvent("change", {});
+        target.splice.apply(target, [0, target.length].concat(source.toArray())).dispose();
       }
```

Comment:
no, target is `qx.data.Array` and `.splice` returns a new instance of `qx.data.Array` which needs to be disposed